cves/CVE-2016-4438.yml

```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
+  An attacker can execute code in the application. It is possible by passing 
+  a malicious expression. If it didn't work a denial of service would ensue.
```

Comment:
Some more detail in the description might be nice. How is this expression passed to struts? What problem in the code allows the remote code execution?

Reply:
I updated this on the last commit

Reply:
Are you able to find an example of a malicious input? The unit tests may be enlightening.

Reply:
I searched but couldn't find any.